packages/jest-cli/src/__tests__/generateEmptyCoverage.test.js

```diff
@@ -35,14 +38,34 @@ module.exports = {
 };`;
 
 it('generates an empty coverage object for a file without running it', () => {
+  const coverageMap = istanbulCoverage.createCoverageMap({});
+  const sourceMapStore = libSourceMaps.createSourceMapStore();
+  const rootDir = '/tmp';
+  const filepath = path.join(rootDir, './sum.js');
+
   const emptyCoverage = generateEmptyCoverage(
     src,
-    '/sum.js',
+    filepath,
     makeGlobalConfig(),
     makeProjectConfig({
       cacheDirectory: os.tmpdir(),
-      rootDir: os.tmpdir(),
+      rootDir,
+      transform: [
+        ['^.+\\.js$', path.join(__dirname, '../../../babel-jest/src/index.js')],
+      ],
     }),
   );
-  expect(emptyCoverage && emptyCoverage.coverage).toMatchSnapshot();
+
+  expect(typeof emptyCoverage).toBe('object');
+
+  let coverage = emptyCoverage && emptyCoverage.coverage;
+
+  if (emptyCoverage && emptyCoverage.sourceMapPath) {
```

Comment:
The result from `generateEmptyCoverage` is a [coverage data and an optional sourceMapPath](https://github.com/facebook/jest/blob/master/packages/jest-cli/src/generateEmptyCoverage.js#L15). If an sourceMapPath is returned, jest will use it to [transform the coverage data](https://github.com/facebook/jest/blob/master/packages/jest-cli/src/reporters/coverage_reporter.js#L179).

Reply:
The actually coverage for this piece of code is in [this commit](https://github.com/facebook/jest/pull/7388/commits/d85408846d2472c0ca8be08325e32018b4d63045) and [looks like this](https://github.com/facebook/jest/pull/7388/commits/d85408846d2472c0ca8be08325e32018b4d63045#diff-a90f0003922b543487fb515aafc83faaR130)

```
      ...
      "4": Object {
        "end": Object {
          "column": -1,
          "line": null,
        },
        "start": Object {
          "column": 4,
          "line": 8,
        },
      },
      "5": Object {
        "end": Object {
          "column": -1,
          "line": null,
        },
        "start": Object {
          "column": 0,
          "line": 12,
        },
      ...
```